examples/sysctrl/hello-sysctrl/examples/init_hpc.rs

```diff
@@ -99,22 +101,68 @@ fn main() -> ! {
 
     print_example_name!();
 
+    // Enable SDRAM
+    let ddr_mode = 0b1 << 1;
+    let axi_enable = 0b1 << 2;
+    sprint!("Enabling SDRAM ddr_mode={:#x}, axi_enable={:#x}...",
+            ddr_mode,
+            axi_enable,
+    );
+    sdram::sdram_cfg_axi_ddr_mode_mask(ddr_mode);
+    sdram::sdram_cfg_axi_enable_mask(axi_enable);
+    sprintln!(" done");
+
     for i in 0..5 {
         let addr = HPC_BOOTRAM_ADDR + i * 4;
         sprint!("Writing 0x6f into {:#x}...", addr);
         unsafe { ptr::write_volatile(addr as *mut _, 0x6f) };
         sprintln!(" done");
     }
 
-    let hpc_core_en = 0xf;
+        // Configure execute regions for SDRAM
```

Comment:
Broken indentation. Do you run a formatter on the code? :)

Reply:
I'll put these for emacs acting up